server/src/types/chats.rs

```diff
@@ -1,12 +1,15 @@
 use log::info;
-use spacetimedb::{table, Identity, ReducerContext, Table, Timestamp};
+use spacetimedb::{table, Identity, ReducerContext, Timestamp};
+use spacetimedsl::dsl;
 
 use crate::types::players::get_username;
 
+#[dsl(plural_name = global_chat_messages)]
 #[table(name = global_chat, public)]
```

Comment:
If the plural name is `global_chat_messages` you should use `global_chat_message` as singular name.